core/store/orm/schema.go

```diff
@@ -95,6 +95,7 @@ type ConfigSchema struct {
 	JobPipelineResultWriteQueueDepth          uint64          `env:"JOB_PIPELINE_RESULT_WRITE_QUEUE_DEPTH" default:"100"`
 	KeeperMaximumGracePeriod                  int64           `env:"KEEPER_MAXIMUM_GRACE_PERIOD" default:"100"`
 	KeeperMinimumRequiredConfirmations        uint64          `env:"KEEPER_MINIMUM_REQUIRED_CONFIRMATIONS" default:"12"`
+	KeeperRegistryGasOverhead                 uint64          `env:"KEEPER_REGISTRY_GAS_OVERHEAD" default:"200000"`
```

Comment:
This gas overhead is ~120k over what it needs to be for the `performUpkeep` overhead. It's not a tremendous amount, but it's not nothing. Is this necessary for `checkUpkeep`? Could we cut it down any closer to the 80k hardcoded in the contract, or what do you think of separating out the two config variables?